### ISSUE TITLE
Do iOS / Android planed to be supported ?

### ISSUE THREAD

**Issue by meiry** [andr3wmac/Torque6#24]
Hello just found this engine , from bgfx git , 
is there any plan to make it work in iOS / Android ?

**Comment by andr3wmac** [comment]
Yep! The engines platform layer is from Torque2D which supports both Android and iOS. The old OpenGL code has been removed and replaced with bgfx. bgfx supports both Android and iOS so it won't take too much work to get Torque 6 running on Android and iOS. 

My biggest concern is the rendering was initially coding primarily with desktop in mind. Deferred shading is too heavy for most mobile devices. I've recently refactored rendering code to prepare for mobile platforms. It's 100% planned but unfortunately I can't give a timeline at the moment.

**Comment by Vanderemit** [comment]
This is fantastic news. Your engine is shaping up to be the first legitimately production capable & ready,  free & open source game engine available that is truly permissively licensed and has an actual fully implemented IDE/Editor. The "blender of game engines", if you will. I sure hope you don't abandon this incredibly auspicious project ever.

**Comment by andr3wmac** [comment]
@Vanderemit Thanks for the kind words. It's got a long way to go but I'm happy with the progress so far. I spend a lot of time working with various engines so the project is very relevant to my career and personal development and I don't see that changing in the near future. Development may slow at times as life becomes hectic but I don't foresee me abandoning the project. Stay tuned :)

**Comment by meiry** [comment]
Any news with the mobile support ?

**Comment by andr3wmac** [comment]
@meiry What platform is at the top of your list? Android or iOS? I may try for one of them this weekend. Even when I get the support in there though the engine still isn't ready to make full games, if you're on a tight timeline there are more complete engine choices out there.

**Comment by meiry** [comment]
Hey im on the search for 3d "light" engine for ios first
